examples/advanced.exs

```diff
@@ -8,4 +8,4 @@ defmodule AdvancedScenario do
   end
 end
 
-Beamchmark.run(AdvancedScenario, duration: 10, delay: 1, output_dir: "/tmp/beamchmark_output")
```

Comment:
Let's leave `output_dir` option but change path to something other. 
I would also show how to specify HTML formatter as this is something that most people will probably be interested in.